VoxelSniperCore/src/main/java/com/github/kevindagame/brush/BiomeBallBrush.java

```diff
@@ -0,0 +1,124 @@
+package com.github.kevindagame.brush;
+
+import com.github.kevindagame.snipe.SnipeData;
+import com.github.kevindagame.util.Messages;
+import com.github.kevindagame.util.VoxelMessage;
+import com.github.kevindagame.voxelsniper.biome.VoxelBiome;
+import com.github.kevindagame.voxelsniper.block.IBlock;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class BiomeBallBrush extends AbstractBrush {
+
+    private VoxelBiome selectedBiome = VoxelBiome.PLAINS;
+
+    @Override
+    public void info(VoxelMessage vm) {
+        vm.brushName(this.getName());
+        vm.size();
+        vm.custom(Messages.SELECTED_BIOME_TYPE.replace("%selectedBiome%", this.selectedBiome.key()));
+    }
+
+    @Override
+    public String getPermissionNode() {
+        return "voxelsniper.brush.biomeball";
+    }
+
+    @Override
+    public void arrow(SnipeData v) {
+        this.biome(v);
+    }
+
+    @Override
+    protected void powder(SnipeData v) {
+        this.biome(v);
+    }
+
+    private void biome(final SnipeData v) {
+        final int brushSize = v.getBrushSize();
+        final double brushSizeSquared = Math.pow(brushSize, 2);
+        var targetBlock = this.getTargetBlock();
+        int blockPositionX = targetBlock.getX();
+        int blockPositionY = targetBlock.getY();
+        int blockPositionZ = targetBlock.getZ();
+        for (int z = 1; z <= brushSize; z++) {
+            final double zSquared = Math.pow(z, 2);
+            this.getWorld().setBiome(blockPositionX + z, blockPositionY, blockPositionZ, this.selectedBiome);
+            this.getWorld().setBiome(blockPositionX + z, blockPositionY, blockPositionZ, this.selectedBiome);
+            this.getWorld().setBiome(blockPositionX - z, blockPositionY, blockPositionZ, this.selectedBiome);
+            this.getWorld().setBiome(blockPositionX, blockPositionY + z, blockPositionZ, this.selectedBiome);
+            this.getWorld().setBiome(blockPositionX, blockPositionY - z, blockPositionZ, this.selectedBiome);
+            this.getWorld().setBiome(blockPositionX, blockPositionY, blockPositionZ + z, this.selectedBiome);
+            this.getWorld().setBiome(blockPositionX, blockPositionY, blockPositionZ - z, this.selectedBiome);
```

Comment:
Don't really like all these repeated calls, but I don't necessarily know we could do much about it ;)